independent-publisher-connectors/AADInvitationManager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,193 @@
+    "version": "1.0",
+    "contact": {
+      "name": "Daniel Laskewitz",
+      "url": "https://youtube.com/daniellaskewitz",
+      "email": "connectorsupport@laskewitz.io"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-1.0"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://privacy.microsoft.com/en-us/privacystatement"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "IT Operations"
+    }
+  ],
+  "host": "graph.microsoft.com",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/v1.0/invitations": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "inviteRedeemUrl": {
+                  "type": "string",
+                  "description": "The URL the user can use to redeem their invitation."
+                },
+                "invitedUserDisplayName": {
+                  "type": "string",
+                  "description": "The display name of the user being invited."
+                },
+                "invitedUserEmailAddress": {
+                  "type": "string",
+                  "description": "The email address of the user being invited."
+                },
+                "sendInvitationMessage": {
+                  "type": "boolean",
+                  "description": "Indicates whether an email should be sent to the user being invited or not."
+                },
+                "invitedUserMessageInfo": {
+                  "type": "object",
+                  "properties": {
+                    "messageLanguage": {
+                      "type": "string",
+                      "description": "The language you want to send the default message in. If the customizedMessageBody is specified, this property is ignored, and the message is sent using the customizedMessageBody. The language format should be in ISO 639. The default is en-US."
+                    },
+                    "ccRecipients": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "emailAddress": {
+                            "$ref": "#/definitions/emailAddress"
+                          }
+                        }
+                      },
+                      "description": "Additional recipients the invitation message should be sent to. Currently only 1 additional recipient is supported."
+                    },
+                    "customizedMessageBody": {
+                      "type": "string",
+                      "description": "customizedMessageBody"
```

Comment:
@Laskewitz All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

Reply:
Fixed